traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -845,39 +770,78 @@ def phys_locations_post_data(to_session: TOSession, request_template_data: list[
 		if not isinstance(name, str):
 			raise TypeError(f"name must be str, not '{type(name)}'")
 		phys_locations["name"] = name[:4] + randstr
-		shortName = phys_locations["shortName"]
+		shortname = phys_locations["shortName"]
```

Comment:
shortName = phys_locations["shortName"]
 	shortname = phys_locations["shortName"]
 	please check name 
 	short_name is recomended